testeng/jobs/backupJenkins.groovy

```diff
@@ -69,18 +69,18 @@ secretMap.each { jobConfigs ->
         // during snapshotting
         configure { project ->
             project / buildWrappers << 'hudson.plugins.execution.exclusive.ExclusiveBuildWrapper' {
-                skipWaitOnRunningJobs false
+                skipWaitOnRunningJobs true
             }
         }
 
         // Run snapshotting script once a day, when there is usually no Jenkins activity
```

Comment:
This is incorrect now that it's once a week, right? Probably best to remove this comment, and put a comment instead above the cron statements below (cron('0 1 * * 6')) that explain them in english (e.g. At 1am on Sundays).